UPGRADE-1.10.md

```diff
@@ -1,12 +1,3 @@
-# UPGRADE FROM `v1.10.0` TO `v1.10.1`
-
-1. API is disabled by default, to enable it you need to set flag to ``true`` in ``app/config/packages/_sylius.yaml``:
```

Comment:
This should be left here IMHO as it's big BC break and should be widely visible.